File: packages/insomnia-httpsnippet/test/fixtures/output/javascript/jquery/query.js
var settings = {
  "async": true,
  "crossDomain": true,
  "url": "http://mockbin.com/har?foo=bar&foo=baz&baz=abc&key=value",
  "method": "GET",
  "headers": {}
}

$.ajax(settings).done(function (response) {
  console.log(response);
});

File: packages/insomnia-httpsnippet/test/fixtures/output/c/libcurl/query.c
CURL *hnd = curl_easy_init();

curl_easy_setopt(hnd, CURLOPT_CUSTOMREQUEST, "GET");
curl_easy_setopt(hnd, CURLOPT_URL, "http://mockbin.com/har?foo=bar&foo=baz&baz=abc&key=value");

CURLcode ret = curl_easy_perform(hnd);
